### ISSUE TITLE
[Feature Request] Add link page

### ISSUE THREAD

**Issue by xseth03** [docmost/docmost#342]
Hi,

Firstly, congratulations on your work. 
Docmost is a fabulous tool and I imagine it will improve considerably over time.

I'd like to suggest an improvement, but I don't think it's been suggested yet, so if you do, please excuse me.

I think it would be great to be able to create a link with the pages in order to build a navigation summary.

Best regards

**Comment by xcraftr** [comment]
I switch from Notion to Docmost and that's the only thing I miss about Notion :
the **/page** feature.

I'll never move back, docmost is amazing : open-source and self-hosted, the best thing ever.
Thanks !

**Comment by enchove** [comment]
Is there any further progress on this topic?
“Sub Page” and ‘Link to Page’ are really, really cool features.

We already use the Sidebar to organize our documents and access them quickly.
But “Sub Page” and ‘Link to Page’ featfure are really intuitive and quick to react to when we need to refer to different Page.

I use the Sidebar for large categorizations, 
And For more detailed articles, I use the link page to organize them so that they are easily distinguishable.

If I were a programmer, I would have applied to docmost at once and supported docmost more powerfully

Thanks

**Comment by cpetzel** [comment]
After getting this setup fairly quickly (great docs), this was the feature that I was mising the most after coming from notion.

**Comment by pickles4evaaaa** [comment]
> After getting this setup fairly quickly (great docs), this was the feature that I was mising the most after coming from notion.
> 
> Thanks for the great project, and hopefully we can get this feature added soon!

Same here. I’m really hoping this gets implemented

**Comment by M-Sviridov** [comment]
I agree, it would be an amazing feature, killer project nevertheless.

I kind of found a workaround in the mean time, you could highlight the text you want to transform to be a link, then choose "Add link" and paste the link of the page you want to redirect to.

It does the job.

Cheers.

**Comment by Philipinho** [comment]
@M-Sviridov, work in progress. See PR: https://github.com/docmost/docmost/pull/604.

**Comment by nsantini** [comment]
Looks like the PR got merged yesterday 🚀

What's the release process like? can't wait to upgrade to the version with this change 💯

**Comment by Philipinho** [comment]
This feature has been introduced in [v0.8](https://github.com/docmost/docmost/releases/tag/v0.8.0).
